src/NexusMods.Abstractions.Library/Jobs/IInstallLoadoutItemJob.cs

```diff
@@ -30,3 +31,15 @@ public interface IInstallLoadoutItemJob : IJobDefinition<LoadoutItemGroup.ReadOn
     /// </summary>
     public ILibraryItemInstaller? Installer { get; }
 }
+
+/// <summary>
+/// The result of installing a loadout item via the <see cref="IInstallLoadoutItemJob"/>.
+/// This struct holds a <see cref="LoadoutItemGroup"/> for the item which was just installed.
+///
+/// If the value is 'null' then the job was attached to an existing, external transaction
+/// to be part of a larger atomic operation.
+/// (Done by passing an <see cref="ITransaction"/> transaction to the job.)
+/// This is because the value is not yet available; as the transaction
+/// needs to be externally committed by the caller.
+/// </summary>
+public record struct InstallLoadoutItemJobResult(LoadoutItemGroup.ReadOnly? LoadoutItemGroup);
```

Comment:
What is the advantage of passing a struct record of a nullable value rather than just passing the nullable value?

Reply:
I wanted to:

- Add documentation to the return type in an explicit manner.
- Avoid excessive refactors if there's ever a need to add more values in the return structure/type.

Mainly the first one. This is actually called `Nominal Typing`, https://nexus-mods.github.io/NexusMods.App/developers/decisions/project/0004-use-nominal-typing/